front/src/Stores/ChatStore.ts

```diff
@@ -33,6 +43,42 @@ function getAuthor(authorId: number): PlayerInterface {
     return author;
 }
 
+function createWritingStatusMessageStore() {
+    const { subscribe, update } = writable<WritingMessage[]>([]);
+    return {
+        subscribe,
+        addWritingStatus(authorId: number, status: number) {
```

Comment:
If I'm right, we can restrict the type to only allowed values.

```suggestion
        addWritingStatus(authorId: number, status: 5 | 6) {
```